python/mlc_chat/compiler/extern/extern.py

```diff
@@ -71,3 +76,7 @@ def configure(rope_scale: float, rope_theta: float) -> None:
             rope_scale=rope_scale,
             rope_theta=rope_theta,
         )
+    if store.fastertransformer is not None:
+        assert store.target.kind.name == "cuda"
+        assert tvm.get_global_func("relax.ext.cutlass", True)
```

Comment:
add an error message saying something like tvm is not compiled against cutlass, please set USE_CUTLASS when compiling tvm